packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -281,6 +281,9 @@ function BlockSelectionButton( { clientId, rootClientId } ) {
 	);
 
 	const dragHandleLabel = __( 'Drag' );
+	const showBlockDraggable =
+		( canMove && editorMode === 'navigation' ) ||
+		( editorMode === 'zoom-out' && canMove && ! isBlockTemplatePart );
```

Comment:
```suggestion
	const showBlockDraggable = canMove && 
		( editorMode === 'navigation' || ( editorMode === 'zoom-out' && ! isBlockTemplatePart ) );
```

Reply:
Debatable? 😁

Reply:
```suggestion
	const showBlockDraggable =
		( canMove && editorMode === 'navigation' ) ||
		( canMove && editorMode === 'zoom-out' && ! isBlockTemplatePart );
```

Changing the order also makes it more readable.

Reply:
In general, I think all the inline conditions below could be brought up here to make things more readable too.